### DISCUSSION TITLE
Construa um algoritmo que mostre a mensagem "Alo mundo" na tela em python. urgente eu não consigo fazer to a horas

### DISCUSSION THREAD

**Discussion by mario-baianor** [community/community#49443] [Codespaces] [answered]
### Select Topic Area

Question

### Body

repito construa um algoritmo que mostre a mensagem "Alo mundo" na tela em python. urgente eu não consigo fazer to a horas

**Comment by danisantosss** [comment, marked as answer]
print("Alô mundo")

**Comment by jxaovito** [comment]
print("Hello World")

**Comment by jxaovito** [comment]
print("Hello World")

**Comment by LucasKlemke** [comment]
print("Alô mundo")
